### ISSUE TITLE
Review dark_current step 

### ISSUE THREAD

**Issue by stscijgbot-rstdms** [spacetelescope/romancal#486]
_Issue [RCAL-335](https://jira.stsci.edu/browse/RCAL-335) was created on JIRA by [Nadia Dencheva](https://jira.stsci.edu/secure/ViewProfile.jspa?name=dencheva):_

Review the dark_current step and validate the code and results.

**Comment by stscijgbot-rstdms** [comment]
_Comment by [Paul Huwe](https://jira.stsci.edu/secure/ViewProfile.jspa?name=phuwe) on [JIRA](https://jira.stsci.edu/browse/RCAL-335?focusedCommentId=593945#comment-593945):_

This is implemented in RCAL PR-349: #520

**Comment by stscijgbot-rstdms** [comment]
_Comment by [Megan Sosey](https://jira.stsci.edu/secure/ViewProfile.jspa?name=sosey) on [JIRA](https://jira.stsci.edu/browse/RCAL-335?focusedCommentId=594625#comment-594625):_

The algorithms page that INS is maintaining: <https://innerspace.stsci.edu/display/RI/DMS+Pipeline+Algorithms>

**Comment by stscijgbot-rstdms** [comment]
_Comment by [David Davis](https://jira.stsci.edu/secure/ViewProfile.jspa?name=ddavis) on [JIRA](https://jira.stsci.edu/browse/RCAL-335?focusedCommentId=594631#comment-594631):_

We're waiting on details from INS on how to implement the error propagation in the dark step which is likely to involve an additional error array to be added to the total error. This will involve updates to RAD, roman_datamodels and RCAL. So once we have more details we will open a new ticket to address those three updates and will close this ticket.
